dashboard/models.py

```diff
@@ -219,7 +219,8 @@ def json(self, include_id=True):
 
 
 class File(models.Model):
-    id = models.BigIntegerField(primary_key=True, verbose_name="File Id")
+    resource_type = models.TextField(verbose_name="Resource Type")
```

Comment:
resource_type can probably just be a CharField as it's going to be small. Also will change in the migration.